sdk/keyvault/Azure.Security.KeyVault.Administration/api/Azure.Security.KeyVault.Administration.netstandard2.0.cs

```diff
@@ -1,51 +1,66 @@
 namespace Azure.Security.KeyVault.Administration
 {
-    public partial class BackupOperation : Azure.Operation<System.Uri>
+    public partial class BackupOperation : Azure.Operation<Azure.Security.KeyVault.Administration.BackupResult>
     {
         public BackupOperation(Azure.Security.KeyVault.Administration.KeyVaultBackupClient client, string id) { }
-        public System.DateTimeOffset? EndTime { get { throw null; } }
         public override bool HasCompleted { get { throw null; } }
         public override bool HasValue { get { throw null; } }
         public override string Id { get { throw null; } }
         public System.DateTimeOffset? StartTime { get { throw null; } }
-        public override System.Uri Value { get { throw null; } }
+        public override Azure.Security.KeyVault.Administration.BackupResult Value { get { throw null; } }
         public override Azure.Response GetRawResponse() { throw null; }
         public override Azure.Response UpdateStatus(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public override System.Threading.Tasks.ValueTask<Azure.Response> UpdateStatusAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public override System.Threading.Tasks.ValueTask<Azure.Response<System.Uri>> WaitForCompletionAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public override System.Threading.Tasks.ValueTask<Azure.Response<System.Uri>> WaitForCompletionAsync(System.TimeSpan pollingInterval, System.Threading.CancellationToken cancellationToken) { throw null; }
+        public override System.Threading.Tasks.ValueTask<Azure.Response<Azure.Security.KeyVault.Administration.BackupResult>> WaitForCompletionAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public override System.Threading.Tasks.ValueTask<Azure.Response<Azure.Security.KeyVault.Administration.BackupResult>> WaitForCompletionAsync(System.TimeSpan pollingInterval, System.Threading.CancellationToken cancellationToken) { throw null; }
+    }
+    public partial class BackupResult
+    {
+        internal BackupResult() { }
+        public System.Uri backupFolderUri { get { throw null; } }
+        public System.DateTimeOffset EndTime { get { throw null; } }
```

Comment:
I thought we were going to leave those on operation? Makes more sense that way.

Reply:
Do you think it should exist in both places? For `RestoreOperation` I wanted to put something in there rather than make it an empty class.

Reply:
I don't really have any strong feelings one way or the other. I think having a class that only returns a URI is fine. Maybe we never expand it, maybe we don't. But it's consistent. We don't (or shouldn't, given what architects thought) have any other models that are directly from the BCL.